src/resources/views/checkout/payment.blade.php

```diff
@@ -44,7 +44,13 @@
                         </template>
                     </div>
                     <div class="checkout__terms">
-                        <label><input type="checkbox" v-model="currentCheckout.user.terms">@lang('Please check the box to agree to our') <a href="/pages/legal/terms-conditions-summary" target="_blank">terms &amp; conditions</a></label>
+                        <label>
+                            <input type="checkbox" v-model="currentCheckout.user.terms">
+                            @lang('Please check the box to agree to our')
+                            <a class="checkout__terms-link" href="{{ route('termsConditions') }}" target="_blank">
```

Comment:
@m2de, are you happy with using lower camel-case for the route naming convention?

Reply:
Whatever we currently do?

Reply:
Currently we use lower camel-case @m2de, but just thought I'd double check.